airbyte/mcp/_local_ops.py

```diff
@@ -234,6 +234,84 @@ def read_source_stream_records(
         return records
 
 
+# @app.tool()  # << deferred
+def get_stream_previews(
+    source_connector_name: Annotated[
```

Comment:
We generally just call this 'source_name', being a subset of all connector_names.